### ISSUE TITLE
1280x720 windowed resolution support.

### ISSUE THREAD

**Issue by CortexReaver** [mefistotelis/swars#15]
Hello, there is seems have been no updates lately, but I've seen a pretty fresh commit by the developer and want to propose some more resolution modes, such as 1280x720 and maybe even higher. Game can only run at 320x200 in window mode, which is infinitesimal small by today's monitors standards, even 640x400 is not enough for a comfortable experience. But something like 1280x720 windowed is bare minimum and will be enough if nothing else is possible to implement.

I have an idea to stream this game live, but have faced some difficulties capturing it... Not to say that full screen mode screws up your colors and two monitor setup completely.

**Comment by mefistotelis** [comment]
> Game can only run at 320x200 in window mode,

Not true. Actually it never does, unless you disable stretching by parameters.

>  even 640x400 is not enough for a comfortable experience

In high-res mode, the current version of the game (v0.3.3) runs at 640x480, only movies are 640x400.
The original port (v0.3.0) always uses 640x480.


Anyway, I may either do higher res or not, haven't decided yet.
The simplest way to expand screen size right now is to use higher value for "minimal physical resolution".
Currently it is set to:
```
LbScreenSetMinPhysicalScreenResolution(400);
```

Which basically means 320x200 is doubled, 640x400 or 640x480 is left intact. Set that to 720 and recompile the game, and any resolution lower that 720p will be multiplied until it reaches the value. Pixels will get doubled, tripled, quadrupled etc. on screen.

Many people stream the game; but they tend to use DOS version, not the port.

**Comment by CortexReaver** [comment]
> Not true. Actually it never does, unless you disable stretching by parameters.

Yeah, I can't quite remember how it was. Think I first tried your port first, the commands for window mode not worked, was it -w or something, and then I switched back to older vexillium version. But you got the main point overall.

> Set that to 720 and recompile the game

Sorry, not a coder myself.

> Many people stream the game; but they tend to use DOS version, not the port.

Don't have a DOS machine, and hate to use emulators. Have not tried yet to run original game under windows 7. But will it run on win 10? doubt it.

Thanks for the answer anyway :)

**Comment by mefistotelis** [comment]
> Sorry, not a coder myself.

If you're afraid of becoming one, I can assure you that changing one number within a text file is not enough to get there.

Related issue:
https://github.com/mefistotelis/swars/issues/14

**Comment by mefistotelis** [comment]
This was somewhat implemented, so closing.
